docs/ADVANCED.md

```diff
@@ -147,4 +147,62 @@ Please note that `Push.Configure` is called automatically when using the `config
     // Direct access to the send functions
     Push.sendAPN(userToken, options);
     Push.sendGCM(userTokens, options)
-```
\ No newline at end of file
+```
+
+### Send API
+
+You can send push notifications from the client or the server using Push.send(). If sending from the client you are required to use [allow/deny](ADVANCED.md#client-security)) rules.
+
+There are 4 required parameters that must be passed to `Push.send`. They are:
+* `from` : reserved for future use. intended to be internally used by gcm to generate a collapse key. this can be any random string at the moment
+* `title` : the bold title text that is displayed in the notification
+* `text` : the normal sub-text that is displayed in the notification
+* a selection query from below
+
+The 4th parameter is a selection query for determining who the message should be sent to. This query can be one of the three following items:
+* `query` : {} or {userId : 'XXXXX'} or {id : 'XXXXX'} 
```

Comment:
if possible id also do `` around the json objects. like so:
`{}` or `{userId : 'XXXXX'}` or `{id : 'XXXXX'}`